frontend/simple/views/UserProfileView.vue

```diff
@@ -1,26 +1,252 @@
 <template>
-  <vue-assistant :views="views" @done="done"></vue-assistant>
+  <section class="section full-screen">
+    <div class="columns">
+      <div class="column is-one-third"></div>
+      <div class="column" >
+        <form ref="ProfileForm"
+              name="ProfileForm"
+              @submit.prevent="save"
+              data-vv-scope="ProfileForm"
+        >
+          <p
+                  class="notification is-success has-text-centered"
+                  v-if="profileSaved"
+                  id="ProfileSaveSuccess"
+          >
+            <i class='notification-icon fa fa-check'></i>
+            <i18n>Profile saved successfully!</i18n>
+          </p>
+          <p
+                  class="notification is-danger has-text-centered"
+                  v-if="errorMsg"
+          >
+            <i class='notification-icon fa fa-exclamation-triangle'></i>
+            {{errorMsg}}
+          </p>
+          <div class="panel">
+            <p class="panel-heading">
+              <i18n>Profile Attributes</i18n>
+            </p>
+            <div class="panel-block">
+              <figure class="media-left">
+                <p class="image is-64x64">
+                  <img v-bind:src="$store.getters.currentUserIdentityContract.attributes.picture">
+                </p>
+              </figure>
+              <div class="media-content">
+                <strong><i18n>Profile Picture</i18n>:</strong> <input class="input" type="text" v-validate data-vv-rules="url" name="profilePicture" v-model="edited.picture" placeholder="http://">
+                <i18n v-if="errors.has('ProfileForm.profilePicture')" class="help is-danger">
+                  The profile picture must be a valid url
```

Comment:
OK, the default URL is a relative URL, and ironically fails the validation because of that. Needs to be made into a real URL, i.e. absolute fully-qualified URL.

![screen shot 2017-05-30 at 6 39 44 pm](https://cloud.githubusercontent.com/assets/138706/26612111/89c4e63a-4567-11e7-923c-9ceb40243626.jpg)